src/components/UserList.js

```diff
@@ -1,23 +1,49 @@
 import React from 'react';
 import UserItem from './UserItem';
+import { addNewUser, removeEntireList } from "./actions/actions";
+import { connect } from "react-redux";
 
 class UserList extends React.Component {
-    render() {
-        return (
-            <>
-                <form>
-                    <div>
-                        <input />
-                        <input type="submit" value="dodaj" />
-                    </div>
-                </form>
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.props.onAdd(e.target.children[0].children[0].value);
+    e.target.children[0].children[0].value = "";
```

Comment:
W React staramy się nie działać na DOM. Dlatego tutaj można zrobić "Reactowy" `state`, który przechowuje informacje wpisane do pola input i przekazujemy je przez np. `this.state.user`. Nie wszystko musimy trzymać w reduxie. Możemy spokojnie wykorzystywać to co poznaliśmy do tej pory.